### DISCUSSION TITLE
how to calculate error percentage for alerts

### DISCUSSION THREAD

**Discussion by deep-iimjobs** [SigNoz/signoz#4993] [Q&A] [answered]
[https://signoz.io/docs/alerts-management/metrics-based-alerts/#example](url)

Can someone tell what metric to take as A so that, for B = "signoz_calls_total", Function F1 = A*100/B gives the error percentage.
please refer the gif at the end of the page.

@pranay01 @pranayrauthu

**Comment by nitishfy** [comment]
Hi @deep-iimjobs , can you please reshare the link?

**Reply by deep-iimjobs** [reply, marked as answer]
Hi, I have found the solution in the updated documentation. Thank you.

[https://signoz.io/docs/alerts-management/metrics-based-alerts/](https://signoz.io/docs/alerts-management/metrics-based-alerts/)
